cmd/traffic_cache_tool/CacheTool.cc

```diff
@@ -182,6 +197,16 @@ Stripe::Chunk::clear()
 
 Stripe::Stripe(Span *span, Bytes start, CacheStoreBlocks len) : _span(span), _start(start), _len(len)
 {
+  const char *diskPath        = span->_path.path();
+  const size_t hash_seed_size = strlen(diskPath);
+  const size_t hash_text_size = hash_seed_size + 32;
+  char *hash_text             = static_cast<char *>(ats_malloc(hash_text_size));
+  strncpy(hash_text, diskPath, hash_text_size);
+  snprintf(hash_text + hash_seed_size, (hash_text_size - hash_seed_size), " %" PRIu64 ":%" PRIu64 "", (uint64_t)_start,
+           (uint64_t)_len.count() * STORE_BLOCK_SIZE);
```

Comment:
This should just be `_len`. If not, we'll need to update the version of Scalar here.